numpy/core/multiarray.py

```diff
@@ -31,8 +31,8 @@
     'count_nonzero', 'c_einsum', 'datetime_as_string', 'datetime_data',
     'dot', 'dragon4_positional', 'dragon4_scientific', 'dtype',
     'empty', 'empty_like', 'error', 'flagsobj', 'flatiter', 'format_longfloat',
-    'frombuffer', 'fromfile', 'fromiter', 'fromstring', 'inner',
-    'interp', 'interp_complex', 'is_busday', 'lexsort',
+    'frombuffer', 'fromfile', 'fromiter', 'fromstring', 'get_handler_name',
```

Comment:
```suggestion
    'frombuffer', 'fromfile', 'fromiter', 'fromstring', 'get_handler_name',
```
I think we should maybe add an additional word to make "handler" explicit, since this is top-level.

Reply:
Nevermind, it is only in `core`, so I don't mind the name (although maybe "allocator" or so would be more clear anyway)

Reply:
Let's leave this for a future PR? I am really bad with names. I didn't want to use "allocator" since it is really a "allocator/free policy handler", and ended up with "handler"